cmd/notes-from-governance-rex.txt

```diff
@@ -0,0 +1,38 @@
+Rakuten Logo + "Open Source" is ダメ
```

Comment:
i think these are internal notes and don't need to be here, do they?

- [ ] remove internal notes